updated change logs with sso deletion information (#3000)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/changelog.mdx

# Changelog

### 2026-07-17

> **HCP Access Management:** HCP organizations will now have all SSO users removed upon deletion of SSO connection. Suspending SSO connection will not delete users connected to the organization through SSO. 

Reviewer: It looks like the date ordering is now off:

<img width="1073" height="620" alt="Image" src="https://github.com/user-attachments/assets/b91599ea-3ec6-42b3-96f8-040ff4c39f56" />

Author: I noticed that when I created my branch from main. Let me fix the ordering real quick

# Changelog

### 2026-07-17

> **HCP Access Management:** HCP organizations will now have all SSO users removed upon deletion of SSO connection. Suspending SSO connection will not delete users connected to the organization through SSO. 

Reviewer 2: ```suggestion
**HCP Access Management:** When an HCP organization's SSO connection is deleted, HCP will now remove all SSO users from the organization. Suspending an SSO connection will not result in HCP removing SSO users from the organization.
```

I'm not 100% sure on the terminology here, whether the user's are truly deleted vs. "removed", I defer to the engineering team on that.

Reviewer: The users will be permanently deleted not just removed from the organization. This is an important distinction thank you for mentioning it.

Reviewer: We should be consistent with our terminology here. We use "removed" for the deletion but "not deleted" for suspension.
